src/components/Company/Industry/Edit.tsx

```diff
@@ -1,22 +1,28 @@
 import { useFormContext } from 'react-hook-form'
 import { useTranslation } from 'react-i18next'
+import { useMemo } from 'react'
 import { useIndustryItems } from './Context'
-import { ComboBox } from '@/components/Common'
+import { ComboBoxField } from '@/components/Common'
 
 export interface IndustryFormFields {
   naics_code: string
 }
 
 export const Edit = () => {
   const { t } = useTranslation('Company.Industry')
-  const { control } = useFormContext<IndustryFormFields>()
   const { items } = useIndustryItems()
 
+  const options = useMemo(() => {
+    return items.map(item => ({
+      label: item.name,
+      value: item.id,
+    }))
+  }, [items])
```

Comment:
Looking closer at the code, it seems we might be able to improve performance here by just using `label`/`value` naming in the generated items over in IndustrySelect here

https://github.com/Gusto/embedded-react-sdk/blob/f2a9966a3ced9a02d94ff8e17a1694df306e3942/src/components/Company/Industry/IndustrySelect.tsx#L27-L32

Which would eliminate the need for this mapping

Reply:
Good catch, that is very logical and also proof we picked the right API that we're deleting code.